### PR TITLE
LIT7104Yegbaru

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
A new qene, minor corrections.

### CODE REVIEW

**Comment by eu-genia** [comment]
Please do not create new works, there is again a problem with numbers.

File: new/LIT7150Yegbaru.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7150Yegbaru" xml:lang="en" type="work">
```
Comment (eu-genia) [inline new/LIT7150Yegbaru.xml:4]: nb I have renamed the new file to LIT7150Yegbaru. update the ms record where needed

**Review by eu-genia — APPROVED** [review]

**Comment by eu-genia** [comment]
> Please do not create new works, there is again a problem with numbers.

should be back to normal

File: new/LIT7104Yegbaru.xml [hunk 2]
```diff
@@ -0,0 +1,93 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7104Yegbaru" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ይግበሩ፡ በኁባሬ፡ ተዝካረ፡ ሰንብት፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Yǝgbaru ba-ḫubare tazkāra sanbat (Qǝne of śǝllāse-type)</title>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT7104Yegbaru.xml:9]: ḫubāre

Comment (CarstenHoffmannMarburg) [reply new/LIT7104Yegbaru.xml:9]: śǝllāse type (without hyphen as above)

File: new/LIT7104Yegbaru.xml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                </listRelation>
+                
+                <div type="edition">
+                    <div type="textpart" xml:lang="gez">
+                        <note>The text is taken from <ref type="mss" corresp="BNFabb145"/>, fol. 56vb-vc 
+                            (division into verses is indicated with the four dot asterisk ።; the end of the poem is indicated with the double four dot asterisk ። ።).</note>
+                        <ab>                                                  
+                            <l n="1">ይግበሩ፡ በኁባሬ፡ ተዝካረ፡ ሰንብት፡ ሁከተ፡ ዓውሎ፡ ቀላያተ፡ ሙሴ፡ ወአሮን፡ ሕዘበ፡ ፳ኤል፡ ኵሎሙ።</l> 
+                            <l n="2">ማዕከለ፡ ኤርትራ፡ አሐቲ፡ ቤቶሙ።</l>
+                            <l n="3">ሚካኤል፡ በዕብሬትከ፡ ቀዲሙ።</l>
```
Comment (karljonaskarlsson) [inline new/LIT7104Yegbaru.xml:69]: ሚካኤል፡ ሊቅ

File: new/LIT7104Yegbaru.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/LIT7104Yegbaru.xml:9]: No. The proper orthography is with hyphen.

File: new/LIT7069ItyopyaAbbay.xml [hunk 4]
```diff
@@ -32,11 +32,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p>This <term key="Qene">qǝne</term> is possibly of <term key="Kwellekomu">mawaddǝs za-kʷǝllǝkǝmu type</term>. 
+                <p>This <term key="Qene">qǝne</term> is possibly of <term key="Kwellekomu">mawaddǝs za-kʷǝllǝkǝmu-type</term>. 
                     Neither author nor type are indicated in the manuscript. The identy of Walda Giyorgis mentioned in the poem is unclear.
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/LIT7069ItyopyaAbbay.xml:36]: ok

File: new/LIT7104Yegbaru.xml [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7104Yegbaru" passive="BNFabb145"/>
+                </listRelation>
+                
+                <div type="edition">
+                    <div type="textpart" xml:lang="gez">
+                        <note>The text is taken from <ref type="mss" corresp="BNFabb145"/>, fol. 56vb-vc
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/LIT7104Yegbaru.xml:64]: :)

File: new/LIT7104Yegbaru.xml [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7104Yegbaru" passive="BNFabb145"/>
+                </listRelation>
+                
+                <div type="edition">
+                    <div type="textpart" xml:lang="gez">
+                        <note>The text is taken from <ref type="mss" corresp="BNFabb145"/>, fol. 56vb-vc 
+                            (division into verses is indicated with the four dot asterisk ።; the end of the poem is indicated with the double four dot asterisk ። ።).</note>
+                        <ab>                                                  
+                            <l n="1">ይግበሩ፡ በኁባሬ፡ ተዝካረ፡ ሰንብት፡ ሁከተ፡ ዓውሎ፡ ቀላያተ፡ ሙሴ፡ ወአሮን፡ ሕዘበ፡ ፳ኤል፡ ኵሎሙ።</l>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/LIT7104Yegbaru.xml:67]: :)

File: new/LIT7104Yegbaru.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/LIT7104Yegbaru.xml:69]: ok

**Pull request merged**